examples/Mediawiki/TemplateTransclusion.yaml

```diff
@@ -28,32 +28,8 @@ tests:
           path: api.php
           parameters:
             action: parse
-            page: Template:APIParsing
-            format: json
-        response:
-          body: !pcre/pattern: /Testing template transclusion/
-      # Edit a template
-      - request:
-          method: post
-          path: api.php
-          parameters:
-            action: edit
+            text: "{{APIParsing}}"
             title: Template:APIParsing
```

Comment:
title should be some other page's title, like Foo

Reply:
Done.